internal/exec/engine.go

```diff
@@ -338,7 +338,7 @@ func (e *Engine) signalNeedNet() error {
 	if f, err := os.Create(e.NeedNet); err != nil {
 		return err
 	} else {
-		f.Close()
+		_ = f.Close()
```

Comment:
internal/exec/engine: Ignore close() error 
bc3c2892c37d660511f5d123a9d1125598a9eb55

Missing lint details in commit message.